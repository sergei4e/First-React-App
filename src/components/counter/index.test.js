import React from 'react';
import Counter from './';
import dom from 'react-test-renderer';

const renderTree = dom.create(<Counter posts = { [9, 6, 7, 23] } />).toJSON();

test('Counter should correspond to its snapshot', () => {
    expect(renderTree).toMatchSnapshot();
});
