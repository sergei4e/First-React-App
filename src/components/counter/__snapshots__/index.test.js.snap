// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter should correspond to its snapshot 1`] = `
<section
  className={undefined}
>
  Posts count: 
  4
</section>
`;
